podman-rpm-info-vars.sh

```diff
@@ -2,10 +2,10 @@
 
 # Set to "dev" to pull from the podman-next copr, set to "release"
 # to pull the ext rom from the fedora build system based of the versions below.
-export PODMAN_RPM_TYPE="dev"
+export PODMAN_RPM_TYPE="release"
```

Comment:
This change has to be reverted before we merge on main. `release` should only be on the release branches. I added it to only show the rpm being replaced with the one from copr.

I can do the revert after I have LGTMs.